### ISSUE TITLE
Unable to run shell command (/bin/sh): fork/exec /bin/sh: no such file or directory

### ISSUE THREAD

**Issue by cheveyo20** [gravitational/teleport#1988]
**What happened**:
The moment i try to login into the web-ui ssh terminal, i keep getting the following error:
ssh: could not start shell

**What you expected to happen**:
a standard shell :)

**How to reproduce it (as minimally and precisely as possible)**:
I followed the official documentation "Getting Started" and setup a cluster

**Environment**:
- Teleport version (use `teleport version`): 2.6.1
- Tsh version (use `tsh version`):
- OS (e.g. from /etc/os-release): ubuntu 16.04

**Browser environment**
Vivaldi (Chrome based)

**Relevant Debug Logs If Applicable**
```
ERRO [NODE]      Unable to run shell command (/bin/sh): fork/exec /bin/sh: no such file or directory id:3x
 local:10.11.47.6:3022 login:pt remote:10.50.22.97:58613 teleportUser:pt srv/sess.go:680                 x
ERRO             fork/exec /bin/sh: no such file or directory regular/sshserver.go:1110 
```
- tsh --debug
- teleport --debug

**Comment by klizhentas** [comment]
it's probably the way you start your teleport server  - the process can't fork/exec. Can you show me your unit file or how you start shell?

**Comment by cheveyo20** [comment]
I just launched it via SuperPutty (SSH) as sudo. I also tried it to launch it inside a tmux session :)

**Comment by moritzdietz** [comment]
I just ran into this myself.
The problem occurred for me when I ran on the shell like so `sudo teleport start --config /etc/teleport.yaml `.
Then I stumbled over this open issue and checked whether the problem could be fixed if I started teleport via the systemd unit file provided in the example folder. Sure enough the problem does not occur when started using the systemd unit file.
Posting this in case anybody runs into the same problems and is looking for a solution.
